core/vm/actor/builtin/types/miner/monies.hpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+#include <vm/actor/builtin/states/miner/miner_actor_state.hpp>
+#include "common/outcome.hpp"
+#include "common/smoothing/alpha_beta_filter.hpp"
+#include "const.hpp"
+#include "primitives/types.hpp"
+#include "vm/actor/builtin/types/miner/policy.hpp"
+#include "vm/runtime/runtime.hpp"
+#include "vm/version/version.hpp"
+
+namespace fc::vm::actor::builtin::types::miner {
+  using fc::common::smoothing::FilterEstimate;
+  using fc::primitives::BigInt;
+  using fc::primitives::ChainEpoch;
+  using fc::primitives::StoragePower;
+  using fc::primitives::TokenAmount;
+  using fc::vm::actor::builtin::states::MinerActorState;
+  using fc::vm::actor::builtin::types::miner::VestSpec;
+  using fc::vm::version::NetworkVersion;
+
+  class Monies {
+   public:
+    int precommit_deposit_factor = 20;
+    int initial_pledge_factor = 20;
+    ChainEpoch precommit_deposit_projection_period =
+        ChainEpoch(precommit_deposit_factor) * fc::kEpochsInDay;
+    ChainEpoch initial_pledge_projection_period =
+        ChainEpoch(initial_pledge_factor) * fc::kEpochsInDay;
+    ChainEpoch termination_lifetime_cap = ChainEpoch(70);
+    // v0
+    virtual outcome::result<TokenAmount> ExpectedRewardForPower(
```

Comment:
```suggestion
    virtual outcome::result<TokenAmount> expectedRewardForPower(
```

Reply:
Usually we use function names in camel case notation starting by small letter. Please change all function names